### ISSUE TITLE
[Wav2Vec2.0] Diversity Loss if quite Different in Train and Eval set

### ISSUE THREAD

**Issue by LEECHOONGHO** [pytorch/fairseq#3804]
![image](https://user-images.githubusercontent.com/44384060/130043557-bd47170f-bda5-4f22-9f81-d6721546067c.png)

Hello, I'm pre-training my custom Wav2Vec2.0
But in training progress, I found a problem in diversity loss.

Diversity loss's train loss is 0.04~0.1, but eval loss is about 0.81. 
Is it normal that usage of codebook is quite unstable(?) in early stage of training?

Number of eval data is about 18000, batch size is 15, and I'm using noisy utterances crawled from youtube.
